keyboards/dichotomy/keymaps/beat/keymap.c

```diff
@@ -0,0 +1,111 @@
+#include QMK_KEYBOARD_H
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _MOUSE 8
+#define _ADJUST 16
```

Comment:
Non-consecutive layer numbers will generate extra empty layers in between, unnecessarily increasing flash usage. The better way to define layers is with an enum, eliminating the need to number them manually:

```suggestion
enum layer_names {
    _QWERTY,
    _LOWER,
    _RAISE,
    _MOUSE,
    _ADJUST
};
```

Reply:
Good point. I updated the PR